src/newrelic-provider.ts

```diff
@@ -42,11 +58,24 @@ const defaults: NewRelicProviderOptions = {
     debug: false
 }
 
+function preload_newrelic(plugin) {
+    /*
+    const self = this;
+    console.log('here', this);
+    console.log(plugin);
+    */
+}
+
+export const extras = {
+    preload: preload_newrelic
+}
+
 
 Object.assign(NewrelicProvider, {defaults})
 
 export default NewrelicProvider
 
 if ('undefined' !== typeof (module)) {
```

Comment:
preload will need to be a property of the plugin definition function anyway, so no need to export separately

Reply:
OK